src/canmatrix/cli/compare.py

```diff
@@ -23,20 +23,22 @@
 
 from __future__ import absolute_import, division, print_function
 
-import logging
 import sys
+import logging
 import typing
-from builtins import *
 
 import click
 
-import canmatrix.compare
+import canmatrix.compare as cancompare
+
+import canmatrix.log as canmatrix_log
+
 
 logger = logging.getLogger(__name__)
 
 
 @click.command()
-@click.option('-v', '--verbose', 'verbosity', help="Output verbosity", count=True, default=1)
+@click.option('-v', '--verbose', 'verbosity', help="Output verbosity", count=True, default=False)
```

Comment:
is this working ? 

a count-option starting with "False"?

Reply:
Sorry, this will be revert.
Not using count-option before